### ISSUE TITLE
Question inquiry

### ISSUE THREAD

**Issue by xiaoshier-good** [ph-w2000/DuoCast#4]
Hello, this paper has a huge workload and is a very good one. However, when conducting experiments on the Shanghai dataset, I encountered the following issues:
1. What is the appropriate training epoch?
2. Is CSI-M the evaluation criterion for selecting the best weight?

**Comment by ph-w2000** [comment]
Thank you for your interest in our work.

1.  The appropriate training epoch depends on the batch size and learning rate. In practice, the appropriate training epoch is selected based on validation performance. We usually train the model until the validation metrics become stable, and then select the checkpoint with the best validation performance.
2. Yes, CSI-M is the main evaluation criteria for selecting the best weight. In some cases, we also consider SSIM together with CSI-M, especially when two checkpoints have very close CSI-M values, to ensure the selected model also preserves reasonable spatial structure.
